jina/drivers/evaluate.py

```diff
@@ -0,0 +1,58 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+from typing import Iterable
+from .helper import DocGroundTruthPair
```

Comment:
`Groundtruth` can be used as one word without capital `T` https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=groundtruth&btnG=